src/lib/client.ts

```diff
@@ -63,6 +63,11 @@ export interface ClientData {
   farming: ClientFarming
 }
 
+export interface ClientIdentity {
+  public_key: string,
```

Comment:
It is non-idiomatic to use `snake_case` in TypeScript, I would suggest to add `#[serde(rename_all = "snake_case")]` to the struct on backend.

Reply:
Changing this (line 67) to `publicKey` and changing where it is used:
[src/pages](https://github.com/subspace/subspace-desktop/pull/32/files#diff-e879b9701cf110ae7a5636e48ba2803b2f6a8b808b3539e2937607f7a0200d11R265)
`farmerIdentity.public_key` into `farmerIdentity.publicKey` breaks some functionality. 
The desktop app cannot retrieve the solutions that have been found by the farmer and thus displays **0 solutions found**, although it finds solutions...

Don't know the cause yet. Also tried to change other `public_key` namings to `publicKey`, still did not work. I believe it has to match with tauri-side variable naming. Not sure though